packages/eslint-plugin/rules/no-wp-process-env.js

```diff
@@ -0,0 +1,93 @@
+const NAMES = new Set(
+	/** @type {const} */ ( [
+		'GUTENBERG_PHASE',
+		'IS_GUTENBERG_PLUGIN',
+		'IS_WORDPRESS_CORE',
+		'SCRIPT_DEBUG',
```

Comment:
Technically I think `SCRIPT_DEBUG` was never part of `process.env`, but I don't think its harmful to include it.

Reply:
Yes, when `SCRIPT_DEBUG` was introduced we knew that `process.env` is no longer a way forward.